keyboards/skme/zeno/info.json

```diff
@@ -8,6 +8,11 @@
         "pid": "0x0001",
         "device_version": "0.0.1"
     },
+    "matrix_pins": {
```

Comment:
After merging, this causes issues -- seems like this file has already been migrated on `develop`?

See https://github.com/qmk/qmk_firmware/blob/develop/keyboards/skme/zeno/info.json.

Reply:
Looks like #20023.